src/__tests__/Cell.spec.js

```diff
@@ -100,10 +104,10 @@ describe('Cell Tests', () => {
     it('should append the update cell class to the dom node if present and cell is updated', () => {
       var updateClass = 'highlight-cell';
       testProps.column.getUpdateCellClass = () => updateClass;
-      var cellInstance = TestUtils.renderIntoDocument(<Cell {...testProps}/>);
+      var cellInstance = new PropTestUtils(Cell, testProps);
       // force update
       cellInstance.setProps({rowData: {}, selectedColumn: testProps.column});
```

Comment:
since setProps is being deprecated there is not point having a test for this. Instead we should simulate the element being updated which can be done with ReactDOM.Render on the same container node 
https://facebook.github.io/react/blog/2015/10/01/react-render-and-top-level-api.html